docs/resources/grant_database_role.md

```diff
@@ -69,13 +69,13 @@ resource "snowflake_grant_database_role" "g" {
 
 ### Required
 
-- `database_role_name` (String) The fully qualified name of the database role which will be granted to share or parent role.
+- `database_role_name` (String) The fully qualified name of the database role which will be granted to share or parent role. For more information about this resource, see [docs](https://registry.terraform.io/providers/Snowflake-Labs/snowflake/latest/docs/resources/database_role).
```

Comment:
Done. These links were not used in data sources. I suppose we can add resource <-> data source links, but this can be done in a separate PR. WDYT? cc @sfc-gh-jcieslak